src/assets/styles/_variables.scss

```diff
@@ -31,6 +31,7 @@
 
   // MISC
   --TextGradientBackground: linear-gradient(90deg, #473cfb 0%, #a300bd 33.17%);
+  --StepBackground: #169fcc;
```

Comment:
I just noticed this, but how come we are using CapitalCasing for these? Traditionally CSS vars are kebab-case and the majority of them above are. Kinda looks like this CapitalCasing is influenced by old Kongponents, but it seems like even they are moving towards `kui-kebab-case`?

Anyway just a comment here that is a super nit we should probably go back to kebab-case for CSS vars. I guess there might be a lint rule for that if we wanted to be lint-y about it.

1 sec gonna approve this now, no action here.

Reply:
Yeah, I think we can rename these at some point.